frontend/src/components/Sidebar/Sidebar.styling.tsx

```diff
@@ -38,10 +38,21 @@ const StyledLink = styled(NavLink)`
     }
 `;
 
+const Accordion = styled('button')`

```

Comment:
Is there a reason we are styling a button and naming it accordion? 
Why would it be in the Sidebar (headerbar) ?